annotation-file-utilities/src/annotator/find/Insertion.java

```diff
@@ -389,6 +485,20 @@ public static void decorateType(List<Insertion> innerTypeInsertions, final Type
         decorateType(innerTypeInsertions, outerType, null);
     }
 
+    /**
+     * Adds each of the given inner type insertions to the correct part of the
+     * type, based on the insertion's type path.
+     *
+     * @param innerTypeInsertions
+     *          The insertions to add to the type. These must be inner type
+     *          insertions, meaning each of the insertions' {@link Criteria}
+     *          must contain a {@link GenericArrayLocationCriterion} and
+     *          {@link GenericArrayLocationCriterion#getLocation()} must return
+     *          a non-empty list.
+     * @param outerType The type to add the insertions to.
```

Comment:
`@param` text should not end with punctuation (or start with a capitalized letter), if it is a single sentence fragment.